game-core/src/test/java/games/strategy/engine/framework/startup/ui/PlayerTypeTest.java

```diff
@@ -32,6 +35,7 @@ void newPlayerWithName() {
 
     stream(PlayerType.values())
```

Comment:
side-note, this test assertion is suspect, contains logic. It might be better to have 4 or 5 relatively duplicated lines that asserts for each expected play type and hardcodes the expected values. It would be easy to understand and extend and avoids having logic in the assertion conditions.